libr/core/cmd_search.c

```diff
@@ -170,17 +170,12 @@ static void cmd_search_init(RCore *core) {
 	DEFINE_CMD_DESCRIPTOR_SPECIAL (core, /x, slash_x);
 }
 
-static int search_hash(RCore *core, const char *hashname, const char *hashstr, ut32 minlen, ut32 maxlen) {
+static int search_hash(RCore *core, const char *hashname, const char *hashstr, ut32 minlen, ut32 maxlen, struct search_parameters *param) {
```

Comment:
why not use the short camelcase name?

Reply:
That decision predates me. This is a previously existing struct.

Reply:
oh i see there's no cameltoe case for this struct